### PR TITLE
Get azure to install and import the previously built wheel

### PR DESCRIPTION
**Pull request by hardbyte**
Adds a step to Azure Pipeline to try installing, importing and printing the version of `anonlink` from the previously built platform specific wheel.

Note the `--universal` was incorrect, so has been removed.

It is probably possible and would be very useful to run the pytest suite against the wheel - but it might not do coverage properly so thought I'd leave it out for now.

### CODE REVIEW

File: .azurePipeline/python_init_package_install_steps.yml [hunk 1]
```diff
@@ -14,13 +14,18 @@ steps:
...
-- script: python setup.py sdist bdist_wheel --universal
+- script: python setup.py sdist bdist_wheel
   env:
     CC: ${{ parameters.compiler }}
-  displayName: 'Package'
+  displayName: 'Build Package'
+
+- bash: |
+      pip install dist/anonlink*.whl
+      python -c "import anonlink; print(anonlink.__version__)"
```
Comment (gusmith) [inline .azurePipeline/python_init_package_install_steps.yml]: Should we also run this command in the following step? (i.e. `Install anonlink for testing`)

File: .azurePipeline/python_init_package_install_steps.yml [hunk 2]
```diff
@@ -14,13 +14,18 @@ steps:
       pip install -r requirements.txt
   displayName: 'Install Python requirements'
 
-- script: python setup.py sdist bdist_wheel --universal
+- script: python setup.py sdist bdist_wheel
```
Comment (gusmith) [inline .azurePipeline/python_init_package_install_steps.yml]: We are publishing in Azure some of these build artifacts. Any reason not to build a `universal` one?

Comment (hardbyte) [reply .azurePipeline/python_init_package_install_steps.yml]: `universal` means pure python - anonlink contains compiled code

File: .azurePipeline/python_init_package_install_steps.yml (hunk 1 above)
Comment (hardbyte) [reply .azurePipeline/python_init_package_install_steps.yml]: I don't see why, the `pytest` is much more comprehensive testing that seeing if anonlink is importable (although of course that has to be true).

**Comment by hardbyte** [comment]
I haven't made any changes, but I'll rebase as master has changed

**Pull request merged**